ui/Rhino/AGS/dev/AGS_form_from_layer_cmd.py

```diff
@@ -37,8 +38,10 @@ def RunCommand(is_interactive):
         return
 
     form = FormDiagram.from_graph(graph)
+    base = BaseDiagram.from_graph(graph)
 
     scene.add(form, name='Form', layer='AGS::FormDiagram')
+    scene.add(base, name='Base', layer='AGS::BaseDiagram')
 
```

Comment:
It is annoying to have this diagram drawn at the beginning. Can we somehow turn it visibility off such as the user only starts to see it after it turns on in the display settings:

Somehow this should be added somewhere....
```
base.artist.settings['show.vertices'] = False
base.artist.settings['show.edges'] = False
```

Maybe the `scene.add()` has an option `scene.add(display=False)` since we want to add to the scene but not display it. @brgcode do you have a comment on this?

Reply:
This replicates for all cmd `form_from_...`

Reply:
the base.artist could update the default settings to all False?